core/src/main/java/org/elasticsearch/search/aggregations/metrics/stats/Stats.java

```diff
@@ -51,13 +51,6 @@
     double getSum();
 
     /**
-     * @return The number of values that were aggregated as a String.
-     * @deprecated use String.valueOf(getCount()) instead if the count is needed as a string
-     */
-    @Deprecated
-    String getCountAsString();
```

Comment:
We would need to add this deprecated method to ParsedStats too, and I am not sure what it would return. I don't think it is worth the trouble. That is why I removed it, it was not necessary to deprecate it in the first place.